Woof/Woof/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -1,6 +1,15 @@
 {
   "colors" : [
     {
+      "color" : {
```

Comment:
Why do you need this color here?

Reply:
Remove this, I used this to change color for selected tab, make this via code.